executor/change.go

```diff
@@ -0,0 +1,67 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Missing `// Copyright 2019 PingCAP, Inc.`. You can take other files as an example.

Reply:
finished